scripts/cleanup_edomain_templates.py

```diff
@@ -62,7 +62,7 @@ def is_ovirt_engine_running(rhevm_ip, sshname, sshpass):
         return False
 
 
-def change_edomain_state(api, state, edomain):
+def change_edomain_state(api, state, edomain, provider):
```

Comment:
Is ```provider``` just a string?
Is api from mgmtsystem? What about passing in the provider object where we can get both api and the name from in one object?